python/python/tests/utils/utils.py

```diff
@@ -227,6 +227,34 @@ def generate_lua_lib_code(
     return code
 
 
+def create_lua_lib_with_long_running_function(
```

Comment:
I  think we have the same in another PR - you may have conflicts

Reply:
No.. I've seen in a some PR with docs added.
Nevermind.
Can you add (copy) docs please?